staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -624,6 +624,29 @@ type GlusterfsVolumeSource struct {
 	ReadOnly bool `json:"readOnly,omitempty" protobuf:"varint,3,opt,name=readOnly"`
 }
 
+// Represents a Glusterfs mount that lasts the lifetime of a pod.
+// Glusterfs volumes do not support ownership management or SELinux relabeling.
+type GlusterfsPersistentVolumeSource struct {
+	// EndpointsName is the endpoint name that details Glusterfs topology.
+	// More info: https://releases.k8s.io/HEAD/examples/volumes/glusterfs/README.md#create-a-pod
+	EndpointsName string `json:"EndpointsName" protobuf:"bytes,1,opt,name=EndpointsName"`
```

Comment:
this must be a copy/paste of the field from GlusterfsVolumeSource

Reply:
```
EndpointsName string `json:"endpoints" protobuf:"bytes,1,opt,name=endpoints"`
```

Reply:
Sure, it was experimented in latest patch set to get rid of violation , now reverted the change back to reflect above.